series.py

```diff
@@ -1 +1,42 @@
+"""This module defines functions that implement mathematical series."""
 
+
```

Comment:
Great re-use of code here to have `fibonacci` and `lucas` call your `sum_series` function.